src/services/services.js

```diff
@@ -0,0 +1,67 @@
+import firebase, { db } from './firebase';
+
+export const isCorrectEmail = (email) => {
+  // eslint-disable-next-line no-useless-escape
+  const regExp = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return regExp.test(email);
+};
+
+export const isCorrectPassword = (password) => {
+  // eslint-disable-next-line no-useless-escape
+  if (password.length < 8) {
+    return false;
+  }
+  return true;
+};
+
+export const loginFirebase = async (email, password) => {
+  try {
+    const response = await firebase.auth().signInWithEmailAndPassword(email, password);
+    return response;
+  } catch (error) {
+    throw new Error('Check your login and password');
+  }
+};
+
+export const registerNewUserFirebase = async (email, password) => {
+  try {
+    const response = await firebase.auth().createUserWithEmailAndPassword(email, password);
+    return response;
+  } catch (error) {
+    throw new Error('An error occured :(');
+  }
+};
+
+export const logoutFirebase = async () => {
+  try {
+    await firebase.auth().signOut();
+  } catch (error) {
+    throw new Error('An error occured during logout. Try later.');
+  }
+};
+
+export const getDataFirebase = async (collection, field) => {
+  const docRef = db.collection(collection).doc(field);
+  return docRef
+    .get()
+    .then((doc) => {
+      if (doc.exists) {
+        return doc.data();
+      }
+      // doc.data() will be undefined in this case
+      console.log('No such document!');
+      return null;
+    })
+    .catch((error) => {
+      console.log('Error getting document:', error);
+    });
+};
+
+export const getUserStatus = async () => {
+  firebase.auth().onAuthStateChanged((user) => {
+    if (user) {
+      return true;
+    }
+    return false;
```

Comment:
Can we simplify it ? It can be `return Boolean(user)`.. but you also can handle it directly in component.
Don't need to mark method as `async` if you do not use `await`

Reply:
thx, i changed this.
```suggestion
    export const getUserStatus = async () => {
  await firebase.auth().onAuthStateChanged((user) => Boolean(user));
};
```

Reply:
I guess you can even do like this `await firebase.auth().onAuthStateChanged(Boolean)`